templates/template.yaml

```
@@ -136,15 +136,38 @@ Parameters:
     Description: Comma-delimited list of subnets to deploy Fargate tasks in. This parameter must be set if the DeployVpc parameter is "false", otherwise it is ignored.
     Type: CommaDelimitedList
     Default: ""
+  DeployWebUI:
+    Description: Specify whether you would like a Web User Interface deployed. (AuthMethod must be Cognito if WebUI is deployed).
```

Try to keep wording like with the other params 
```suggestion
    Description: Deploy the Web User Interface for this solution.. (AuthMethod must be Cognito if WebUI is set to true).
```